atlas/rules/cobinning.smk

```diff
@@ -204,16 +204,16 @@ rule run_vamb:
     benchmark:
         "logs/benchmarks/vamb/run_vamb.tsv"
     params:
-        params="-m 2000 ", #TODO: I don't know what this is for
-        minfasta=config["cobining_min_contig_length"],
-        separator= config['cobinning_separator']
+        mincontig=config["cobining_min_contig_length"], # skip contigs < 2kb
```

Comment:
Great thank you

Reply:
`minfasta` shall be a minimum bin size, which shall adopte the value of `cobining_min_bin_size`. 2MB may be comparable to the `metabat` bin, smaller bins could be plasmids/phage..